tests/cpp/test_multidevice_pipeline.cpp

```diff
@@ -42,25 +43,6 @@
 
 namespace nvfuser {
 
-namespace {
```

Comment:
Is this intended?

Reply:
No :) Agent was bad at dealing with merge conflicts